charts/atlantis/templates/statefulset.yaml

```diff
@@ -42,6 +42,7 @@ spec:
       {{- end }}
       {{- end }}
       serviceAccountName: {{ template "atlantis.serviceAccountName" . }}
+      shareProcessNamespace: {{ .Values.shareProcessNamespace }}
```

Comment:
Please also bump the minor version on the Chart.yaml file

Reply:
done @nitrocode

Reply:
I believe you bumped the patch version instead of the minor version.